CoEpi.xcworkspace/xcshareddata/swiftpm/Package.resolved

```diff
@@ -0,0 +1,142 @@
+{
```

Comment:
I'm not sure why these changes appear in your PRs. What Xcode version are you using?

Reply:
Version 11.5 (11E608c)

The files have been removed.

Reply:
For what it's worth, I'm on the same version and see those files pop up every time, too.

Reply:
Same version here and I don't. Will investigate.